Lets Do This/Networking/DSOUserManager.m

```diff
@@ -123,15 +123,16 @@ - (void)loadActiveMobileAppCampaignSignupsForUser:(DSOUser *)user completionHand
 
 - (void)endSessionWithCompletionHandler:(void (^)(void))completionHandler errorHandler:(void(^)(NSError *))errorHandler {
     [[DSOAPI sharedInstance] logoutWithCompletionHandler:^(NSDictionary *responseDict) {
-        [SSKeychain deletePasswordForService:[[DSOAPI sharedInstance] northstarBaseURL] account:@"Session"];
-        [SSKeychain deletePasswordForService:[[DSOAPI sharedInstance] northstarBaseURL] account:@"UserID"];
-        [self deleteAvatar];
-        self.user = nil;
-        self.isCurrentUserSync = NO;
+        [self deleteCurrentUser];
         if (completionHandler) {
             completionHandler();
         }
     } errorHandler:^(NSError *error) {
+        NSLog(@"error code within endsession: %ld, error localized description: %@", (long)error.code, error.localizedDescription);
```

Comment:
We don't need this here, `DSOAPI` has its [own method to log errors](https://github.com/DoSomething/LetsDoThis-iOS/blob/fd39a68ca7ac1038fedd04dc3e4844f7b5687dfd/Lets%20Do%20This/Networking/DSOAPI.m#L166) that's basically doing this